From: Adam Conrad <adconrad@ubuntu.com>
Date: Wed, 28 Dec 2016 19:55:34 +0800
Subject: Don't assume /boot/config-* exists, test first.

---
 dkms_common.postinst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dkms_common.postinst b/dkms_common.postinst
index 5102005..79de370 100644
--- a/dkms_common.postinst
+++ b/dkms_common.postinst
@@ -33,6 +33,7 @@ _is_kernel_name_correct() {
     KERNEL_NAME=$1
 
     for kernel in /boot/config-*; do
+        [ -f "$kernel" ] || continue
         KERNEL=${kernel#*-}
         if [ "${KERNEL}" = "${KERNEL_NAME}" ]; then
             CORRECT="yes"
