From ee06dc33b39cb61b1613831d9234183374550d3f Mon Sep 17 00:00:00 2001
From: Alex Tu <alex.tu@canonical.com>
Date: Tue, 30 Jul 2019 15:08:04 +0800
Subject: [PATCH] fix: OBSOLETE_BY be blocked by --force (Closes #89)

---
 dkms | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: dkms/dkms
===================================================================
--- dkms.orig/dkms
+++ dkms/dkms
@@ -780,7 +780,7 @@
         local -a my=(${1//-/ })
         local obsolete=0
         if [[ ${obs} && ${my} ]]; then
-            if [[ $(VER ${obs}) == $(VER ${my}) && ! $force ]]; then
+            if [[ $(VER ${obs}) == $(VER ${my}) ]]; then
                 # They get obsoleted possibly in this kernel release
                 if [[ ! ${obs[1]} ]]; then
                     # They were obsoleted in this upstream kernel
@@ -792,7 +792,7 @@
                     # They were obsoleted in this ABI bump of the kernel
                     obsolete=1
                 fi
-            elif [[ $(VER ${my}) > $(VER ${obs}) && ! $force ]]; then
+            elif [[ $(VER ${my}) > $(VER ${obs}) ]]; then
                 # They were obsoleted in an earlier kernel release
                 obsolete=1
             fi
