Description: Add versioned provides
Forwarded: https://github.com/dell/dkms/pull/107
Author: Martijn Grendelman <Martijn.Grendelman@isaac.nl>
Bug-Debian: https://bugs.debian.org/943484
Last-Update: 2019-10-25

--- dkms-2.7.1.orig/template-dkms-mkdeb/debian/control
+++ dkms-2.7.1/template-dkms-mkdeb/debian/control
@@ -7,5 +7,6 @@ Standards-Version: 3.8.1
 
 Package: DEBIAN_PACKAGE-dkms
 Architecture: DEBIAN_BUILD_ARCH
+Provides: DEBIAN_PACKAGE-modules (= MODULE_VERSION)
 Depends: dkms (>= 1.95), ${misc:Depends}
 Description: DEBIAN_PACKAGE driver in DKMS format.
