Description: Adapt to new bash-completion directory
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Bug-Debian: https://bugs.debian.org/912849
Last-Update: 2019-01-02
Forwarded: https://github.com/dell/dkms/pull/71

--- dkms-2.6.1.orig/Makefile
+++ dkms-2.6.1/Makefile
@@ -13,7 +13,7 @@ VAR = $(DESTDIR)/var/lib/dkms
 MAN = $(DESTDIR)/usr/share/man/man8
 INITD = $(DESTDIR)/etc/rc.d/init.d
 LIBDIR = $(DESTDIR)/usr/lib/dkms
-BASHDIR = $(DESTDIR)/etc/bash_completion.d
+BASHDIR = $(DESTDIR)/usr/share/bash-completion/completions
 KCONF = $(DESTDIR)/etc/kernel
 SHAREDIR = $(DESTDIR)/usr/share
 DOCDIR = $(SHAREDIR)/doc/dkms
